packages/browser-wallet/package.json

```diff
@@ -10,8 +10,7 @@
         "watch": "cross-env WATCH=1 yarn build:dev"
     },
     "dependencies": {
-        "@concordium/browser-wallet-api": "0.1.0",
-        "@concordium/browser-wallet-message-hub": "0.1.0",
```

Comment:
Does the not still refer to types from the message hub? (why are you also removing that?)

Reply:
Because it's not supposed to be published? :thinking:

Reply:
They're not going to be released as packages. The references remain in the respective tsconfigs.